cql3/cql_statement.hh

```diff
@@ -118,6 +119,23 @@ public:
     virtual bool is_conditional() const {
         return false;
     }
+
+    enum class cql_statement_type {
+        NO_METADATA,
+        OPT_METADATA,
+        SELECT
+    };
+
+    static bytes cql_statement_type_serialized(cql_statement_type st) {
+        switch (st) {
+            case cql_statement_type::NO_METADATA: return serialized("NO_METADATA");
+            case cql_statement_type::OPT_METADATA: return serialized("OPT_METADATA");
+            case cql_statement_type::SELECT: return serialized("SELECT");
+            default: return serialized("");
```

Comment:
```suggestion
```

When there is no `default` case, the compiler will error if not all cases are handled, for example:
```
./cql3/cql_statement.hh:130:17: error: enumeration value 'SELECT' not handled in switch [-Werror,-Wswitch]
        switch (st) {
```